arkouda/series.py

```diff
@@ -75,28 +77,38 @@ class Series:
 
     Parameters
     ----------
-    ar_tuple : 2-tuple of arkouda arrays with the first being the grouping key(s) and the
-             second being the value. The grouping key(s) will be the index of the series.
+    ar_tuple : tuple, list.
+        2-tuple of arkouda arrays with the first being the grouping key(s) and the
+        second being the value. The grouping key(s) will be the index of the series.
+        This will be deprecated in future releases.
+    index : pdarray
+        an array of indices associated with the data array.
+        If empty, it will default to a range of ints whose size match the size of the data.
+    data : pdarray, Strings, Categorical
+        a 1D array. Must not be None if ar_tuple is not provided.
```

Comment:
should we remove `ar_tuple` from the doc string since we removed it as a named parameter?

Also we should update the accepted types of data and index in docstring to match the `__init__`

Reply:
Done.  Beefed up doc string.